src/data_warehouse/cli/transforms.py

```
@@ -3,95 +1,122 @@
+        result = subprocess.run(cmd, check=True)
+        click.echo(f"DBT completed with exit code: {result.returncode}")
+    except subprocess.CalledProcessError as e:
+        click.echo(f"DBT failed with exit code: {e.returncode}")
+    finally:
+        os.chdir(original_dir)
+
+@transforms.command()
+@click.option('--models', '-m', help='Specific models to test')
+@click.option('--exclude', '-e', help='Models to exclude from testing')
+@click.option('--profile-dir', help='DBT profile directory', default=os.environ.get('DBT_PROFILES_DIR'))
+@click.option('--project-dir', help='DBT project directory', 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)